src/Bicep.LangServer.IntegrationTests/Helpers/TextDocumentParamHelper.cs

```diff
@@ -14,7 +15,7 @@ public static class TextDocumentParamHelper
             {
                 TextDocument = new TextDocumentItem
                 {
-                    LanguageId = LanguageConstants.LanguageId,
+                    LanguageId = PathHelper.HasBicepparamsExension(documentUri.ToUri()) ? LanguageConstants.ParamsLanguageId : LanguageConstants.LanguageId,
```

Comment:
Would an unsaved buffer created as a params file be treated as a bicep template file until it was saved?